pkg/apis/operator/v1alpha1/validation/validation_test.go

```diff
@@ -1204,24 +1204,24 @@ var _ = Describe("Validation Tests", func() {
 							Entry("valid (unset)", nil, BeEmpty()),
 							Entry("valid (fields unset)", &gardencorev1beta1.APIServerLogging{}, BeEmpty()),
 							Entry("valid (verbosity=0)", &gardencorev1beta1.APIServerLogging{
-								Verbosity: pointer.Int32(0),
+								Verbosity: ptr.To(int32(0)),
 							}, BeEmpty()),
 							Entry("valid (httpAccessVerbosity=0)", &gardencorev1beta1.APIServerLogging{
-								HTTPAccessVerbosity: pointer.Int32(0),
+								HTTPAccessVerbosity: ptr.To(int32(0)),
 							}, BeEmpty()),
 							Entry("valid (verbosity>0)", &gardencorev1beta1.APIServerLogging{
-								Verbosity: pointer.Int32(3),
+								Verbosity: ptr.To(int32(3)),
 							}, BeEmpty()),
 							Entry("valid (httpAccessVerbosity>0)", &gardencorev1beta1.APIServerLogging{
-								HTTPAccessVerbosity: pointer.Int32(3),
+								HTTPAccessVerbosity: ptr.To(int32(3)),
 							}, BeEmpty()),
 							Entry("invalid (verbosity<0)", &gardencorev1beta1.APIServerLogging{
-								Verbosity: pointer.Int32(negativeSize),
+								Verbosity: ptr.To(int32(negativeSize)),
```

Comment:
```suggestion
								Verbosity: ptr.To(negativeSize),
```
`negativeSize` is of type `int32`.Similar changes for other occurrence of `negativeSize`.

Reply:
/hold